packages/api/src/hooks/useFiles.ts

```diff
@@ -0,0 +1,10 @@
+import useWebChatAPIContext from './internal/useWebChatAPIContext';
+
+/**
+ * This is a hook that returns the files and setFiles from the context.
+ */
+export default function useFiles(): [{ files: File[]; setFiles: (files: File[]) => void }] {
```

Comment:
The name "file" could be very vague. It could be files from an incoming activity. Or files for sending out.